UITests/BookmarksPanelTests.swift

```diff
@@ -75,6 +74,22 @@ class BookmarksPanelTests: KIFTestCase {
         // Strangely, earlgrey cannot find the label in buddybuild, but passes locally.
         // Using KIF for this check for now.
         EarlGrey.selectElement(with: grey_accessibilityValue("www.google.ca/")).assert(grey_sufficientlyVisible())
+
+        // Go back to the general bookmark folder for next test
+        BrowserUtils.openLibraryMenu(tester())
+        EarlGrey.selectElement(with: grey_accessibilityLabel("Desktop Bookmarks")).perform(grey_tap())
+
+        // Tapping on Bookmarks to get back to initial bookmark panel, need to use coordinates, no id or label for button
+        var bookmarksGoBackButton = CGPoint()
+        if BrowserUtils.iPad() {
+            bookmarksGoBackButton = CGPoint(x: 114.0, y: 252.0)
```

Comment:
same comment as before, if this doesn't work on all devices, we should mention the devices that can run the test

Reply:
@npark-mozilla in this case I could not find a better way to do it. We could go ahead with this solution with a comment about sims and in the meantime open a bug asking for an id or label for that element